src/proto/h1/conn.rs

```diff
@@ -74,11 +73,15 @@ where
         self.io.set_read_buf_exact_size(sz);
     }
 
-    #[cfg(feature = "client")]
     pub(crate) fn set_title_case_headers(&mut self) {
         self.state.title_case_headers = true;
     }
 
+    #[cfg(feature = "client")]
```

Comment:
Is this only needed for the client? Or maybe I got ahead of myself, and adding support for the received request header cases in the server side is a future addition?

Reply:
It's used only in `client::Builder::handshake`.